docs/clients/dotnet/20.10/embedded/README.md

```diff
@@ -0,0 +1,192 @@
+# Embedded client
```

Comment:
We haven't released the embedded client for v20.10 and above, should we remove the docs relating to it?

Reply:
Nice catch - removed in https://github.com/EventStore/documentation/pull/288/commits/ba3f7e7c27574e7c876a5ae749868472ce656f95.